src/navigation/MainTabNavigator.tsx

```diff
@@ -10,13 +10,15 @@ import RenderingVideoScreen from '../screens/Home/RenderingVideoScreen/Rendering
 import ProfileScreen from '../screens/Home/ProfileScreen/ProfileScreen'
 import StepScreens from '../screens/Home/OnboardingScreen/StepScreens'
 import FinalStepScreen from '../screens/Home/OnboardingScreen/FinalStepScreen'
+import RecordScreen from '../screens/Home/RecordScreen/RecordScreen'
 
 const HomeStack = createStackNavigator(
   {
     Home: HomeScreen,
     Onboarding: OnboardingScreen,
     Steps: StepScreens,
-    FinalStep: FinalStepScreen
+    FinalStep: FinalStepScreen,
+    Record: RecordScreen
```

Comment:
Move this to be right below the HomeScreen.
Also update every question in HomeScreen.tsx to go to RecordScreen. 

I will refactor the navigation later since not all these need to be in a stack anymore.